src/components/FoodPantryContainer/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FunctionComponent } from 'react';
+import { FoodPantry } from 'utils/types';
+import FoodPantryCard from 'src/components/FoodPantryCard/index';
+
+interface Props {
+  pantries: FoodPantry[];
+}
+
+const FoodPantryContainer: FunctionComponent<Props> = ({ pantries }) => {
+  return (
+    <div>
```

Comment:
I prefer using React Fragments (`<>` and `</>`) for wrapping components to have less nesting in the rendered DOM, but its not a big deal to leave the divs